tinyman/v1/client.py

```diff
@@ -44,6 +44,7 @@ def prepare_app_optin_transactions(self, user_address=None):
         return txn_group
 
     def prepare_asset_optin_transactions(self, asset_id, user_address=None):
+        assert asset_id is not 0, "Cannot opt into ALGO"
```

Comment:
`is` is misused here. This blog explains it better than me :)
https://adamj.eu/tech/2020/01/21/why-does-python-3-8-syntaxwarning-for-is-literal/

```
# Test output. Python 3.9.5.

>>> asset_id = 0
>>> assert asset_id is not 0
<stdin>:1: SyntaxWarning: "is not" with a literal. Did you mean "!="?
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
```

```suggestion
        assert asset_id != 0, "Cannot opt into ALGO"
```
Can you update it according to the convention?

Reply:
Done, thanks for the review!